crates/driver/src/infra/config/file/mod.rs

```diff
@@ -192,6 +194,25 @@ struct SolverConfig {
     /// Determines whether the `solver` or the `driver` handles the fees
     #[serde(default)]
     fee_handler: FeeHandler,
+
+    /// List of addresses which receives a specific discount for the protocol
+    /// fees
+    #[serde(default)]
+    protocol_fee_discounted_addresses: Vec<ProtocolFeeDiscountedAddresses>,
```

Comment:
With solver driver colocation in mind (i.e. external parties running their own driver) this should probably be managed by the autopilot.
If we handle it in the autopilot we could simply omit the `fee_policies` on the CoW AMM orders and be done with it. This seems doubly important since the autopilot stores which fees should be applied to each order in an auction to later to do some accounting. So if the autopilot thinks there should be a fee applied to the cow amm orders and the driver simply discards them it does not collect enough fees to pay the protocol next week.